Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2005-04-24
Initial Package Version: 1.1.4
Origin: self
Description: Fixes compilation with jdk-1.5.0
Upstream Status:  Not Required

$LastChangedBy$
$Date$


diff -Naur OOo-build-orig/XmlSearch/src/com/sun/xmlsearch/db/VectorBtree.java OOo-build/XmlSearch/src/com/sun/xmlsearch/db/VectorBtree.java
--- OOo-build-orig/XmlSearch/src/com/sun/xmlsearch/db/VectorBtree.java	2000-11-30 12:02:54.000000000 -0600
+++ OOo-build/XmlSearch/src/com/sun/xmlsearch/db/VectorBtree.java	2005-04-19 01:56:30.289021048 -0500
@@ -274,10 +274,10 @@
     return accessBlock(bl.integerAt(index*4));
   }
 
-  public static void assert(boolean assertion) {
+  /*public static void assert(boolean assertion) {
     if (assertion == false)
       System.err.println("assertion failed");
-  }
+  }*/
 
   public int getIteratorBufferLength() {
     return _blockSize + _vecLen*2;
diff -Naur OOo-build-orig/berkeleydb/db-3.2.9.patch OOo-build/berkeleydb/db-3.2.9.patch
--- OOo-build-orig/berkeleydb/db-3.2.9.patch	2004-01-09 07:44:30.000000000 -0600
+++ OOo-build/berkeleydb/db-3.2.9.patch	2005-04-19 01:56:30.317016792 -0500
@@ -484,3 +484,27 @@
   	*)
   	  # Add libc to deplibs on all other systems.
   	  deplibs="$deplibs -lc"
+*** misc/db-3.2.9/java/src/com/sleepycat/db/DbEnv.java 2004-12-22 15:25:42.919098136 +0100
+--- misc/build/db-3.2.9/java/src/com/sleepycat/db/DbEnv.java   2004-12-22 15:26:05.804142962 +0100
+***************
+*** 78,86 ****
+      //
+      /*package*/ void _notify_dbs()
+      {
+!         Enumeration enum = dblist_.elements();
+!         while (enum.hasMoreElements()) {
+!             Db db = (Db)enum.nextElement();
+              db._notify_dbenv_close();
+          }
+          dblist_.removeAllElements();
+--- 78,86 ----
+      //
+      /*package*/ void _notify_dbs()
+      {
+!         Enumeration enumer = dblist_.elements();
+!         while (enumer.hasMoreElements()) {
+!             Db db = (Db)enumer.nextElement();
+              db._notify_dbenv_close();
+          }
+          dblist_.removeAllElements();
+
diff -Naur OOo-build-orig/javaunohelper/com/sun/star/comp/helper/Bootstrap.java OOo-build/javaunohelper/com/sun/star/comp/helper/Bootstrap.java
--- OOo-build-orig/javaunohelper/com/sun/star/comp/helper/Bootstrap.java	2004-03-30 08:23:53.000000000 -0600
+++ OOo-build/javaunohelper/com/sun/star/comp/helper/Bootstrap.java	2005-04-19 01:56:30.000000000 -0500
@@ -220,11 +220,11 @@
         if (null != bootstrap_parameters)
         {
             pairs = new String [ 2 * bootstrap_parameters.size() ];
-            Enumeration enum = bootstrap_parameters.keys();
+            Enumeration enumer = bootstrap_parameters.keys();
             int n = 0;
-            while (enum.hasMoreElements())
+            while (enumer.hasMoreElements())
             {
-                String name = (String)enum.nextElement();
+                String name = (String)enumer.nextElement();
                 pairs[ n++ ] = name;
                 pairs[ n++ ] = (String)bootstrap_parameters.get( name );
             }
diff -Naur OOo-build-orig/javaunohelper/com/sun/star/lib/loader/Loader.java OOo-build/javaunohelper/com/sun/star/lib/loader/Loader.java
--- OOo-build-orig/javaunohelper/com/sun/star/lib/loader/Loader.java	2004-03-30 08:25:21.000000000 -0600
+++ OOo-build/javaunohelper/com/sun/star/lib/loader/Loader.java	2005-04-19 01:56:30.000000000 -0500
@@ -106,9 +106,9 @@
         ClassLoader loader = clazz.getClassLoader();
         Vector res = new Vector();
         try {
-            Enumeration enum = loader.getResources( "META-INF/MANIFEST.MF" );
-            while ( enum.hasMoreElements() ) {
-                res.add( (URL) enum.nextElement() );
+            Enumeration enumer = loader.getResources( "META-INF/MANIFEST.MF" );
+            while ( enumer.hasMoreElements() ) {
+                res.add( (URL) enumer.nextElement() );
             }
             // the jarfile with the com/sun/star/lib/loader/Loader.class
             // per-entry attribute is most probably the last resource in the
diff -Naur OOo-build-orig/jurt/com/sun/star/comp/servicemanager/ServiceManager.java OOo-build/jurt/com/sun/star/comp/servicemanager/ServiceManager.java
--- OOo-build-orig/jurt/com/sun/star/comp/servicemanager/ServiceManager.java	2002-10-30 09:28:39.000000000 -0600
+++ OOo-build/jurt/com/sun/star/comp/servicemanager/ServiceManager.java	2005-04-19 01:56:30.000000000 -0500
@@ -500,10 +500,10 @@
         throws com.sun.star.uno.RuntimeException
     {
         if (eventListener != null) {
-            java.util.Enumeration enum = eventListener.elements();
+            java.util.Enumeration enumer = eventListener.elements();
             
-            while (enum.hasMoreElements()) {
-                XEventListener listener = (XEventListener) enum.nextElement();
+            while (enumer.hasMoreElements()) {
+                XEventListener listener = (XEventListener) enumer.nextElement();
                 listener.disposing(new com.sun.star.lang.EventObject(this));
             }
         }
@@ -732,16 +732,16 @@
     public XEnumeration createContentEnumeration( String serviceName )
                 throws com.sun.star.uno.RuntimeException
     {
-        XEnumeration enum = null;
+        XEnumeration xenum = null;
         
         java.util.Vector serviceList = (java.util.Vector) factoriesByServiceNames.get(serviceName);
         
         if (serviceList != null)
-            enum = new ServiceEnumerationImpl( serviceList.elements() );
+            xenum = new ServiceEnumerationImpl( serviceList.elements() );
         else 
-            enum = new ServiceEnumerationImpl();
+            xenum = new ServiceEnumerationImpl();
             
-        return enum;
+        return xenum;
 	}
 	  
 	/**
@@ -809,11 +809,11 @@
 		/**
 		 * Constructs a new instance with a given enumeration.
 		 * <p>
-		 * @param	enum	is the enumeration which should been wrapped.
+		 * @param	enumer	is the enumeration which should been wrapped.
 		 * @see		com.sun.star.container.XEnumeration
 		 */
-        public ServiceEnumerationImpl(java.util.Enumeration enum) {
-            enumeration = enum;
+        public ServiceEnumerationImpl(java.util.Enumeration enumer) {
+            enumeration = enumer;
         }
             
 		/**
diff -Naur OOo-build-orig/jurt/com/sun/star/comp/typedescriptionmanager/EnumTypeDescription.java OOo-build/jurt/com/sun/star/comp/typedescriptionmanager/EnumTypeDescription.java
--- OOo-build-orig/jurt/com/sun/star/comp/typedescriptionmanager/EnumTypeDescription.java	2001-12-03 05:00:41.000000000 -0600
+++ OOo-build/jurt/com/sun/star/comp/typedescriptionmanager/EnumTypeDescription.java	2005-04-19 01:56:30.000000000 -0500
@@ -136,12 +136,12 @@
 				Field field = fields[nPos];
 				int nMod = field.getModifiers();
 				
-				if (Enum.class.isAssignableFrom( field.getType() ) &&
+				if (com.sun.star.uno.Enum.class.isAssignableFrom( field.getType() ) &&
 					Modifier.isPublic( nMod ) && Modifier.isFinal( nMod ) && Modifier.isStatic( nMod ))
 				{
 					try
 					{
-						values[nLen] = ((Enum)field.get( null )).getValue();
+						values[nLen] = ((com.sun.star.uno.Enum)field.get( null )).getValue();
 						names[nLen]  = field.getName();
 						++nLen;
 					}
diff -Naur OOo-build-orig/jurt/com/sun/star/comp/typedescriptionmanager/TypeDescriptionManager.java OOo-build/jurt/com/sun/star/comp/typedescriptionmanager/TypeDescriptionManager.java
--- OOo-build-orig/jurt/com/sun/star/comp/typedescriptionmanager/TypeDescriptionManager.java	2000-09-18 10:27:52.000000000 -0500
+++ OOo-build/jurt/com/sun/star/comp/typedescriptionmanager/TypeDescriptionManager.java	2005-04-19 01:56:30.000000000 -0500
@@ -236,7 +236,7 @@
 								ret = new InterfaceTypeDescription( this, c );
 							else if (c.isArray())
 								ret = new SequenceTypeDescription( (XTypeDescription)getByClass( c.getComponentType(), bUnsigned, bInterface ) );
-							else if (Enum.class.isAssignableFrom( c ))
+							else if (com.sun.star.uno.Enum.class.isAssignableFrom( c ))
 								ret = new EnumTypeDescription( c );
 //  							else if (com.sun.star.uno.Union.class.isAssignableFrom( c ))
 //  								ret = new UnionTypeDescription( c );
diff -Naur OOo-build-orig/jurt/com/sun/star/lib/uno/protocols/iiop/Marshal.java OOo-build/jurt/com/sun/star/lib/uno/protocols/iiop/Marshal.java
--- OOo-build-orig/jurt/com/sun/star/lib/uno/protocols/iiop/Marshal.java	2001-01-16 12:01:30.000000000 -0600
+++ OOo-build/jurt/com/sun/star/lib/uno/protocols/iiop/Marshal.java	2005-04-19 01:56:30.000000000 -0500
@@ -446,10 +446,10 @@
 		writeObject(typeDescription, object);
 	}
 
-	void writeEnum(Enum enum) throws Exception {
-		if(DEBUG) System.err.println("##### " + getClass().getName() + ".writeEnum:" + enum + " " + enum.getValue());
+	void writeEnum(Enum writeenum) throws Exception {
+		if(DEBUG) System.err.println("##### " + getClass().getName() + ".writeEnum:" + writeenum + " " + writeenum.getValue());
 
-		write_long(enum.getValue());
+		write_long(writeenum.getValue());
 	}
 
 	void writeThrowable(TypeDescription typeDescription, Throwable throwable) throws Exception {
diff -Naur OOo-build-orig/jurt/com/sun/star/lib/uno/protocols/iiop/iiop.java OOo-build/jurt/com/sun/star/lib/uno/protocols/iiop/iiop.java
--- OOo-build-orig/jurt/com/sun/star/lib/uno/protocols/iiop/iiop.java	2001-02-02 03:01:03.000000000 -0600
+++ OOo-build/jurt/com/sun/star/lib/uno/protocols/iiop/iiop.java	2005-04-19 01:56:30.000000000 -0500
@@ -588,13 +588,13 @@
 		Enumeration elements = _flushList.elements();
 		while(elements.hasMoreElements()) {
 			Object objects[] = (Object[])elements.nextElement();
-			Enum enum = (Enum)objects[0];
+			Enum enumer = (Enum)objects[0];
 			Marshal marshal = (Marshal)objects[1];
 
 			int size = marshal.getSize();
 			byte bytes[] = marshal.reset();
 
-			writeMessageHeader(enum.getValue(), size, dataOutput);
+			writeMessageHeader(enumer.getValue(), size, dataOutput);
 			if(DEBUG) System.err.println("##### " + getClass().getName() + ".flush - bytes:" + size);
 
 			dataOutput.write(bytes, 0, size);
diff -Naur OOo-build-orig/jurt/com/sun/star/lib/uno/protocols/urp/Marshal.java OOo-build/jurt/com/sun/star/lib/uno/protocols/urp/Marshal.java
--- OOo-build-orig/jurt/com/sun/star/lib/uno/protocols/urp/Marshal.java	2003-03-26 06:33:00.000000000 -0600
+++ OOo-build/jurt/com/sun/star/lib/uno/protocols/urp/Marshal.java	2005-04-19 01:56:30.000000000 -0500
@@ -293,10 +293,10 @@
 		writedouble(zDouble.doubleValue());
 	}
 
-	void writeEnum(Enum enum) {
-		if(DEBUG) System.err.println("##### " + getClass().getName() + ".writeEnum:" + enum + " " + enum.getValue());
+	void writeEnum(Enum writeenum) {
+		if(DEBUG) System.err.println("##### " + getClass().getName() + ".writeEnum:" + writeenum + " " + writeenum.getValue());
 
-		writeint(enum.getValue());
+		writeint(writeenum.getValue());
 	}
 
 	void writeThrowable(ITypeDescription iTypeDescription, Throwable throwable) {
diff -Naur OOo-build-orig/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java OOo-build/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java
--- OOo-build-orig/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java	2003-03-26 07:40:24.000000000 -0600
+++ OOo-build/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOAddInCreateIterator.java	2005-04-19 01:56:30.000000000 -0500
@@ -200,7 +200,7 @@
         index--;
     }
     
-    public org.openide.WizardDescriptor$Panel current() {
+    public org.openide.WizardDescriptor.Panel current() {
         return panels[index];
     }
     
diff -Naur OOo-build-orig/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java OOo-build/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java
--- OOo-build-orig/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java	2002-09-03 11:25:06.000000000 -0500
+++ OOo-build/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOIDLWizardIterator.java	2005-04-19 01:56:30.000000000 -0500
@@ -207,7 +207,7 @@
    index--;
   }
 
-  public org.openide.WizardDescriptor$Panel current()
+  public org.openide.WizardDescriptor.Panel current()
    {
      return panels[index];
    }
diff -Naur OOo-build-orig/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java OOo-build/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java
--- OOo-build-orig/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java	2003-03-26 07:40:24.000000000 -0600
+++ OOo-build/odk/source/OOSupport/org/netbeans/modules/openoffice/wizard/OOServiceCreateIterator.java	2005-04-19 01:56:30.000000000 -0500
@@ -214,7 +214,7 @@
         index--;
     }
     
-    public org.openide.WizardDescriptor$Panel current() {
+    public org.openide.WizardDescriptor.Panel current() {
         return panels[index];
     }
     
diff -Naur OOo-build-orig/qadevOOo/runner/base/java_complex.java OOo-build/qadevOOo/runner/base/java_complex.java
--- OOo-build-orig/qadevOOo/runner/base/java_complex.java	2003-06-11 11:24:17.000000000 -0500
+++ OOo-build/qadevOOo/runner/base/java_complex.java	2005-04-19 01:56:30.000000000 -0500
@@ -178,9 +178,9 @@
                 java.io.InputStream in = connection.getInputStream();
                 Properties props = new Properties();
                 props.load(in);
-                Enumeration enum = props.keys();
-                while (enum.hasMoreElements()) {
-                    String key = (String)enum.nextElement();
+                Enumeration enumer = props.keys();
+                while (enumer.hasMoreElements()) {
+                    String key = (String)enumer.nextElement();
                     String value = (String)props.get(key);
                     param.put(key.trim(), value.trim());
                 }
diff -Naur OOo-build-orig/qadevOOo/runner/org/openoffice/RunnerService.java OOo-build/qadevOOo/runner/org/openoffice/RunnerService.java
--- OOo-build-orig/qadevOOo/runner/org/openoffice/RunnerService.java	2003-03-31 11:01:51.000000000 -0600
+++ OOo-build/qadevOOo/runner/org/openoffice/RunnerService.java	2005-04-19 01:56:30.000000000 -0500
@@ -252,10 +252,10 @@
             // get Jar file from connection
             java.util.jar.JarFile f = con.getJarFile();
             // Enumerate over all entries
-            java.util.Enumeration enum = f.entries();
+            java.util.Enumeration enumer = f.entries();
 
-            while (enum.hasMoreElements()) {
-                String entry = enum.nextElement().toString();
+            while (enumer.hasMoreElements()) {
+                String entry = enumer.nextElement().toString();
                 if (entry.endsWith(".csv")) {
                     String module = null;
                     String object = null;
diff -Naur OOo-build-orig/qadevOOo/runner/util/DBTools.java OOo-build/qadevOOo/runner/util/DBTools.java
--- OOo-build-orig/qadevOOo/runner/util/DBTools.java	2003-01-27 10:27:05.000000000 -0600
+++ OOo-build/qadevOOo/runner/util/DBTools.java	2005-04-19 01:56:30.000000000 -0500
@@ -673,12 +673,12 @@
         XEnumerationAccess dbContEA = (XEnumerationAccess)
             UnoRuntime.queryInterface(XEnumerationAccess.class, dbContext) ;
 
-        XEnumeration enum = dbContEA.createEnumeration() ;
+        XEnumeration enumer = dbContEA.createEnumeration() ;
 
         out.println("DatabaseContext registered DataSource's :") ;
-        while (enum.hasMoreElements()) {
+        while (enumer.hasMoreElements()) {
             try {
-                DataSourceInfo inf = new DataSourceInfo(enum.nextElement()) ;
+                DataSourceInfo inf = new DataSourceInfo(enumer.nextElement()) ;
                 inf.printInfo(out) ;
             } catch (com.sun.star.container.NoSuchElementException e) {}
             catch (com.sun.star.lang.WrappedTargetException e) {}
@@ -849,4 +849,4 @@
         throws java.sql.SQLException {
         statement.executeUpdate("drop table if exists " + tbl_name);
     }
-}
\ No newline at end of file
+}
diff -Naur OOo-build-orig/qadevOOo/tests/java/ifc/reflection/_XTypeDescriptionEnumerationAccess.java OOo-build/qadevOOo/tests/java/ifc/reflection/_XTypeDescriptionEnumerationAccess.java
--- OOo-build-orig/qadevOOo/tests/java/ifc/reflection/_XTypeDescriptionEnumerationAccess.java	2003-01-27 12:11:20.000000000 -0600
+++ OOo-build/qadevOOo/tests/java/ifc/reflection/_XTypeDescriptionEnumerationAccess.java	2005-04-19 01:56:30.000000000 -0500
@@ -94,13 +94,13 @@
             return;
         }
         try {
-            XTypeDescriptionEnumeration enum = 
+            XTypeDescriptionEnumeration enumer = 
                         oObj.createTypeDescriptionEnumeration(sString, tClass, 
                         TypeDescriptionSearchDepth.INFINITE);
             try {
                 log.println("Got an enumeration.");
                 while (true) {
-                    XTypeDescription desc = enum.nextTypeDescription();
+                    XTypeDescription desc = enumer.nextTypeDescription();
                     i++;
                     log.println("\tdesc name: " + desc.getName()); 
                 }
diff -Naur OOo-build-orig/qadevOOo/tests/java/mod/_dbaccess/SbaXGridControl.java OOo-build/qadevOOo/tests/java/mod/_dbaccess/SbaXGridControl.java
--- OOo-build-orig/qadevOOo/tests/java/mod/_dbaccess/SbaXGridControl.java	2003-05-27 07:37:57.000000000 -0500
+++ OOo-build/qadevOOo/tests/java/mod/_dbaccess/SbaXGridControl.java	2005-04-19 01:56:30.000000000 -0500
@@ -376,7 +376,7 @@
             new Object[]{ new Integer(0) }, new Object[]{ new Integer(1) }});
         tEnv.addObjRelation("Comparer", new Comparator() {
             public int compare(Object o1, Object o2) {
-                return ((Integer)o1).compareTo(o2);
+                return compare((Integer) o1, o2);
             }
             public boolean equals(Object obj) {
              return compare(this, obj) == 0;
diff -Naur OOo-build-orig/qadevOOo/tests/java/mod/_servicemgr.uno/OServiceManager.java OOo-build/qadevOOo/tests/java/mod/_servicemgr.uno/OServiceManager.java
--- OOo-build-orig/qadevOOo/tests/java/mod/_servicemgr.uno/OServiceManager.java	2003-05-27 08:25:05.000000000 -0500
+++ OOo-build/qadevOOo/tests/java/mod/_servicemgr.uno/OServiceManager.java	2005-04-19 01:56:30.000000000 -0500
@@ -162,12 +162,12 @@
                 (XSet.class, oInterface) ;
             XSet set1 = (XSet) UnoRuntime.queryInterface
                 (XSet.class, srvMan) ;
-            XEnumeration enum = set1.createEnumeration();
-            Object srv = enum.nextElement();
+            XEnumeration enumer = set1.createEnumeration();
+            Object srv = enumer.nextElement();
 
             set.insert(srv) ;
 
-            newElement = enum.nextElement();
+            newElement = enumer.nextElement();
 
             XPropertySet xProp = (XPropertySet)UnoRuntime.queryInterface(
                                             XPropertySet.class, oInterface);
diff -Naur OOo-build-orig/qadevOOo/tests/java/mod/_smgr/OServiceManager.java OOo-build/qadevOOo/tests/java/mod/_smgr/OServiceManager.java
--- OOo-build-orig/qadevOOo/tests/java/mod/_smgr/OServiceManager.java	2003-05-27 08:28:36.000000000 -0500
+++ OOo-build/qadevOOo/tests/java/mod/_smgr/OServiceManager.java	2005-04-19 01:56:30.000000000 -0500
@@ -162,12 +162,12 @@
                 (XSet.class, oInterface) ;
             XSet set1 = (XSet) UnoRuntime.queryInterface
                 (XSet.class, srvMan) ;
-            XEnumeration enum = set1.createEnumeration();
-            Object srv = enum.nextElement();
+            XEnumeration enumer = set1.createEnumeration();
+            Object srv = enumer.nextElement();
 
             set.insert(srv) ;
 
-            newElement = enum.nextElement();
+            newElement = enumer.nextElement();
 
             XPropertySet xProp = (XPropertySet)UnoRuntime.queryInterface(
                                             XPropertySet.class, oInterface);
diff -Naur OOo-build-orig/sandbox/com/sun/star/lib/sandbox/ClassContextImpl.java OOo-build/sandbox/com/sun/star/lib/sandbox/ClassContextImpl.java
--- OOo-build-orig/sandbox/com/sun/star/lib/sandbox/ClassContextImpl.java	2002-11-21 10:49:00.000000000 -0600
+++ OOo-build/sandbox/com/sun/star/lib/sandbox/ClassContextImpl.java	2005-04-19 01:56:30.000000000 -0500
@@ -304,10 +304,10 @@
                         m_class_path_jars = class_path_jars;
                     }
                     
-                    java.util.Enumeration enum = m_class_path_jars.elements();
-                    while (enum.hasMoreElements())
+                    java.util.Enumeration enumer = m_class_path_jars.elements();
+                    while (enumer.hasMoreElements())
                     {
-                        ClassContext context = (ClassContext)enum.nextElement();
+                        ClassContext context = (ClassContext)enumer.nextElement();
                         try
                         {
                             xClass = context.loadClass( className );
diff -Naur OOo-build-orig/sj2/stardiv/app/AppletProps.java OOo-build/sj2/stardiv/app/AppletProps.java
--- OOo-build-orig/sj2/stardiv/app/AppletProps.java	2000-09-18 11:54:03.000000000 -0500
+++ OOo-build/sj2/stardiv/app/AppletProps.java	2005-04-19 01:56:30.000000000 -0500
@@ -138,7 +138,7 @@
 		if (System.getProperty("http.proxyHost") != null) {
 			proxyHost.setText(System.getProperty("http.proxyHost"));
 			proxyPort.setText(System.getProperty("http.proxyPort"));
-			HttpClient.proxyPort = Integer.valueOf(System.getProperty("http.proxyPort")).intValue();
+			//HttpClient.proxyPort = Integer.valueOf(System.getProperty("http.proxyPort")).intValue();
 		} 
 		else {
 			proxyHost.setText("");
diff -Naur OOo-build-orig/sj2/stardiv/controller/PropertyEditor.java OOo-build/sj2/stardiv/controller/PropertyEditor.java
--- OOo-build-orig/sj2/stardiv/controller/PropertyEditor.java	2000-09-18 11:54:03.000000000 -0500
+++ OOo-build/sj2/stardiv/controller/PropertyEditor.java	2005-04-19 01:56:30.000000000 -0500
@@ -197,10 +197,10 @@
 
 			m_DefaultList.removeAll();
 			m_List.removeAll();
-	        java.util.Enumeration enum = m_Properties.propertyNames();
-	        while (enum.hasMoreElements())
+	        java.util.Enumeration enumer = m_Properties.propertyNames();
+	        while (enumer.hasMoreElements())
 	        {
-	            String strKey = (String)enum.nextElement();
+	            String strKey = (String)enumer.nextElement();
 
 	            if (isDefaultKey(strKey))
 	            	addToList(m_DefaultList, strKey, m_Properties.getProperty(strKey));
@@ -551,19 +551,19 @@
     {
     	int nMax = 0;
 
-    	java.util.Enumeration enum = props.propertyNames();
-    	while (enum.hasMoreElements())
-    		nMax = Math.max(nMax, ((String)enum.nextElement()).length());
+    	java.util.Enumeration enumer = props.propertyNames();
+    	while (enumer.hasMoreElements())
+    		nMax = Math.max(nMax, ((String)enumer.nextElement()).length());
 
     	return nMax+2;
     }
 
     protected static boolean isDefaultKey(String strKey)
     {
-    	java.util.Enumeration enum = c_DefaultProperties.elements();
-    	while (enum.hasMoreElements())
+    	java.util.Enumeration enumer = c_DefaultProperties.elements();
+    	while (enumer.hasMoreElements())
     	{
-    		if (strKey.equals((String)enum.nextElement()))
+    		if (strKey.equals((String)enumer.nextElement()))
     			return true;
     	}
 
@@ -574,10 +574,10 @@
     {
     	java.util.Properties saveProps = new java.util.Properties();
 
-    	java.util.Enumeration enum = m_Properties.propertyNames();
-    	while (enum.hasMoreElements())
+    	java.util.Enumeration enumer = m_Properties.propertyNames();
+    	while (enumer.hasMoreElements())
     	{
-    		String strKey = (String)enum.nextElement();
+    		String strKey = (String)enumer.nextElement();
 
     		if (!isDefaultKey(strKey))
 	    		saveProps.put(strKey, m_Properties.getProperty(strKey));
@@ -623,9 +623,9 @@
 
 	    	try
 	    	{
-	    		java.util.Enumeration enum = System.getProperties().propertyNames();
-	    		while (enum.hasMoreElements())
-	    			c_DefaultProperties.addElement(enum.nextElement());
+	    		java.util.Enumeration enumer = System.getProperties().propertyNames();
+	    		while (enumer.hasMoreElements())
+	    			c_DefaultProperties.addElement(enumer.nextElement());
 	    	}
 	    	catch (Exception ignore0) {}
 	    }
@@ -656,10 +656,10 @@
     	}
     	catch (Exception ignore2) {}
 
-    	java.util.Enumeration enum = props.propertyNames();
-    	while (enum.hasMoreElements())
+    	java.util.Enumeration enumer = props.propertyNames();
+    	while (enumer.hasMoreElements())
     	{
-    		String propKey = (String)enum.nextElement();
+    		String propKey = (String)enumer.nextElement();
 
     		// if the key is not a default put the new property into the table
     		if (!isDefaultKey(propKey))
diff -Naur OOo-build-orig/sj2/stardiv/controller/StreamObserver.java OOo-build/sj2/stardiv/controller/StreamObserver.java
--- OOo-build-orig/sj2/stardiv/controller/StreamObserver.java	2000-09-18 11:54:04.000000000 -0500
+++ OOo-build/sj2/stardiv/controller/StreamObserver.java	2005-04-19 01:56:30.000000000 -0500
@@ -65,8 +65,7 @@
 import java.util.Observable;
 import java.net.URL;
 import java.net.MalformedURLException;
-import sun.net.ProgressData;
-import sun.net.ProgressEntry;
+import sun.net.ProgressSource;
 
 
 /** 
@@ -79,12 +78,12 @@
 {
 	public StreamObserver()
 	{
-		ProgressData.pdata.addObserver( this );
+		progressSource.beginTracking();
 	}
 
 	public void dispose()
 	{
-		ProgressData.pdata.deleteObserver( this );
+		progressSource.finishTracking();
 	}
 
 	static long nLastTime;
@@ -95,48 +94,47 @@
 			nLastTime = System.currentTimeMillis();
 			
 			String aStatus = null;
-			ProgressEntry pEntry = (ProgressEntry)aArg;
-			switch( pEntry.what )
+			switch( progressSource.getState() )
 			{
-				case ProgressData.CONNECTED:
+				case CONNECTED:
 					{
 						try
 						{
-							URL aURL = new URL( pEntry.label );
+							URL aURL = progressSource.getURL();
 							aStatus = "Connect: Host " + aURL.getHost() + " contacted. Waiting for reply...";
 						}
-						catch( MalformedURLException e )
+						catch( Exception e )
 						{
-							aStatus = "Connect: " + pEntry.label;
+							aStatus = "Connect: " + progressSource.getURL().toString();
 						}
 					}
 				break;
 
-				case ProgressData.UPDATE:
+				case UPDATE:
 					if( System.currentTimeMillis() - nLastTime > 3000 )
 						// Only 1 update / 3s
 						return;
-					if( pEntry.need != 0 )
+					if( progressSource.getExpected() > 0 )
 					{
 						// Java kommt mit dem letzten Byte nicht r"uber
-						aStatus = "Transfered: " + pEntry.label + " - ";
-						int nPercent = (int)(((long)pEntry.read * 100) / pEntry.need);
+						aStatus = "Transfered: " + progressSource.getURL().toString() + " - ";
+						int nPercent = (int)(((long)progressSource.getProgress() * 100) / progressSource.getExpected());
 						aStatus += String.valueOf( nPercent ) + "% of ";
-						if( pEntry.need >= 1000 )
-							aStatus += String.valueOf( pEntry.need / 1000 ) + "K ";
+						if( progressSource.getExpected() >= 1000 )
+							aStatus += String.valueOf( progressSource.getExpected() / 1000 ) + "K ";
 						else
-							aStatus += String.valueOf( pEntry.need );
+							aStatus += String.valueOf( progressSource.getExpected() );
 						aStatus += "bytes.";
 					}
 					else
-						aStatus = "Transfered: " + pEntry.label;
+						aStatus = "Transfered: " + progressSource.getURL().toString();
 				break;
 
-				case ProgressData.NEW:
-					aStatus = "Try: " + pEntry.label;
+				case NEW:
+					aStatus = "Try: " + progressSource.getURL().toString();
 				break;
-				case ProgressData.DELETE:
-					aStatus = "Done: " + pEntry.label;
+				case DELETE:
+					aStatus = "Done: " + progressSource.getURL().toString();
 				break;
 			}
 			if( aStatus != null )
@@ -179,6 +177,7 @@
 			System.out.println( pStatus );
 		}
 	}
+	private ProgressSource progressSource;
 }
 
 
diff -Naur OOo-build-orig/unoil/makefile.pmk OOo-build/unoil/makefile.pmk
--- OOo-build-orig/unoil/makefile.pmk	2002-06-20 11:41:33.000000000 -0500
+++ OOo-build/unoil/makefile.pmk	2005-04-19 01:56:30.000000000 -0500
@@ -77,7 +77,7 @@
 GENJAVAFILES	:= $(foreach,i,$(JAVAFILESLIST) $(GENJAVADIR)$/$(PACKAGE)$/$i)
 GENCLASSFILES	:= $(foreach,i,$(JAVAFILESLIST:b) $(GENCLASSDIR)$/$(PACKAGE)$/$i.class)
 
-JAVAFLAGS		= -target 1.1
+#JAVAFLAGS		= -target 1.1
 # --- Targets ------------------------------------------------------
 
 JAVAFILES=$(GENJAVAFILES)
diff -Naur OOo-build-orig/xmerge/java/org/openoffice/xmerge/ConvertData.java OOo-build/xmerge/java/org/openoffice/xmerge/ConvertData.java
--- OOo-build-orig/xmerge/java/org/openoffice/xmerge/ConvertData.java	2002-03-07 10:45:20.000000000 -0600
+++ OOo-build/xmerge/java/org/openoffice/xmerge/ConvertData.java	2005-04-19 01:56:30.000000000 -0500
@@ -127,8 +127,8 @@
      *           <code>Vector</code> of <code>Document</code> objects.
      */
     public Enumeration getDocumentEnumeration() {
-        Enumeration enum = v.elements();
-		return (enum);
+        Enumeration enumer = v.elements();
+		return (enumer);
 	}
 
 
diff -Naur OOo-build-orig/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/DocumentDeserializerImpl.java OOo-build/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/DocumentDeserializerImpl.java
--- OOo-build-orig/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/DocumentDeserializerImpl.java	2003-05-29 04:03:38.000000000 -0500
+++ OOo-build/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/DocumentDeserializerImpl.java	2005-04-19 01:56:30.000000000 -0500
@@ -155,14 +155,14 @@
      */
     public Document deserialize() throws ConvertException, IOException {
 	log("\nFound the XSLT deserializer");
-	Enumeration enum = cd.getDocumentEnumeration();
+	Enumeration enumer = cd.getDocumentEnumeration();
 	org.w3c.dom.Document domDoc=null;
 	DOMDocument docOut=null;
 	GenericOfficeDocument doc = null;
 	ByteArrayOutputStream baos =null;
 	GenericOfficeDocument sxwDoc = new GenericOfficeDocument("output");
-	while (enum.hasMoreElements()) {       
-		 docOut = (DOMDocument) enum.nextElement();
+	while (enumer.hasMoreElements()) {       
+		 docOut = (DOMDocument) enumer.nextElement();
 	}
 	domDoc = docOut.getContentDOM();	
 	try{
diff -Naur OOo-build-orig/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/PluginFactoryImpl.java OOo-build/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/PluginFactoryImpl.java
--- OOo-build-orig/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/PluginFactoryImpl.java	2002-04-09 07:18:44.000000000 -0500
+++ OOo-build/xmerge/java/org/openoffice/xmerge/converter/xml/xslt/PluginFactoryImpl.java	2005-04-19 01:56:30.000000000 -0500
@@ -183,9 +183,9 @@
 	String ext= ".txt";
 	String mimeType = null;
 	ConverterInfo ci = this.getConverterInfo();
-	Enumeration enum = ci.getDeviceMime();
-	while (enum.hasMoreElements()) {       
-	    mimeType= (String) enum.nextElement();
+	Enumeration enumer = ci.getDeviceMime();
+	while (enumer.hasMoreElements()) {       
+	    mimeType= (String) enumer.nextElement();
 	}
 	try {
 	    props.load(is);
diff -Naur OOo-build-orig/xmlhelp/source/com/sun/star/help/HelpIndexer.java OOo-build/xmlhelp/source/com/sun/star/help/HelpIndexer.java
--- OOo-build-orig/xmlhelp/source/com/sun/star/help/HelpIndexer.java	2004-10-28 06:52:44.000000000 -0500
+++ OOo-build/xmlhelp/source/com/sun/star/help/HelpIndexer.java	2005-04-19 01:56:30.000000000 -0500
@@ -197,13 +197,13 @@
    			table.close( 0 );
 
 			System.out.println( "Indexing..." );
-      		Enumeration enum = _hashDocInfo.elements();
+      		Enumeration enumer = _hashDocInfo.elements();
         	int cut = 0;
-         	while( enum.hasMoreElements() && cut < 100000000 )
+         	while( enumer.hasMoreElements() && cut < 100000000 )
          	{
           		try
             	{
-          			DocInfo info = ( DocInfo ) enum.nextElement();
+          			DocInfo info = ( DocInfo ) enumer.nextElement();
          			String url = info.getURL();
 					if( url == null )
      				{
@@ -421,12 +421,12 @@
 
      	void dump()
       	{
-			Enumeration enum = _hash.keys();
+			Enumeration enumer = _hash.keys();
    			int j = 0;
 			String[] list = new String[ _hash.size() ];
-   			while( enum.hasMoreElements() )
+   			while( enumer.hasMoreElements() )
       		{
-      			list[j++] = ( String ) enum.nextElement();
+      			list[j++] = ( String ) enumer.nextElement();
            	}
 
 			Db table;
@@ -462,12 +462,12 @@
 
      void dumpHelptext()
      {
-		Enumeration enum = _hashHelptext.keys();
+		Enumeration enumer = _hashHelptext.keys();
    		int j = 0;
 		String[] list = new String[ _hashHelptext.size() ];
-   		while( enum.hasMoreElements() )
+   		while( enumer.hasMoreElements() )
       	{
-      		list[j++] = ( String ) enum.nextElement();
+      		list[j++] = ( String ) enumer.nextElement();
         }
 
 		Db table;
diff -Naur OOo-build-orig/xmlhelp/source/com/sun/star/help/HelpTransformer.java OOo-build/xmlhelp/source/com/sun/star/help/HelpTransformer.java
--- OOo-build-orig/xmlhelp/source/com/sun/star/help/HelpTransformer.java	2001-02-07 03:35:47.000000000 -0600
+++ OOo-build/xmlhelp/source/com/sun/star/help/HelpTransformer.java	2005-04-19 01:56:30.000000000 -0500
@@ -14,11 +14,11 @@
 	{
 		String styleName;
 		Hashtable styles;
-		Enumeration enum = defs.getTable().keys();
+		Enumeration enumer = defs.getTable().keys();
 		
-		while( enum.hasMoreElements() )
+		while( enumer.hasMoreElements() )
 		{
-			styleName = ( String ) enum.nextElement();
+			styleName = ( String ) enumer.nextElement();
 			styles = defs.getEffectiveStyle( styleName );
 			
 			Enumeration styleOptions = styles.keys();
@@ -112,11 +112,11 @@
 		while( it.hasNext() )
 		{
 			Hashtable curr;
-			Enumeration enum = (curr = ( Hashtable )it.next()).keys();
-			while( enum.hasMoreElements() )
+			Enumeration enumer = (curr = ( Hashtable )it.next()).keys();
+			while( enumer.hasMoreElements() )
 			{
 				String str;
-				if( result.get( str = ( String ) enum.nextElement() ) == null )
+				if( result.get( str = ( String ) enumer.nextElement() ) == null )
 					result.put( str,curr.get( str ) );
 			}
 		}
@@ -129,11 +129,11 @@
 	{
 		String str;
 		Hashtable a = getEffectiveStyle( "p.def1" );
-		Enumeration enum = a.keys();
+		Enumeration enumer = a.keys();
 		
-		while( enum.hasMoreElements() )
+		while( enumer.hasMoreElements() )
 		{
-			System.out.println( (str = ( String ) enum.nextElement()) + " : " + ((String)a.get(str)) );
+			System.out.println( (str = ( String ) enumer.nextElement()) + " : " + ((String)a.get(str)) );
 		}
 	}
 	
